tests/core/test_simplicialcomplex.py

```diff
@@ -462,7 +462,7 @@ def test_remove_simplex_id(edgelist6):
     assert set(S.edges.members()) == set(edges)
 
 
-def test_remove_simplex_ids_from(edgelist6):
+def test_remove_simplex_ids_from(edgelist6, edgelist2):
```

Comment:
```suggestion
def test_remove_simplex_ids_from(edgelist6, edgelist4):
```

`edgelist2` is actually not failing before the PR, `edgelist4` is